### PR TITLE
feat/AUT-2332: add basic language support to inline CKE editor

### PR DESCRIPTION
**Pull request by meefox**
## What is the purpose of this pull request?

New feature

## Does your PR contain necessary tests?
not affected

## Did you follow the CKEditor 4 code style guide?

Your code should follow the guidelines from the [CKEditor 4 code style guide](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md) which helps keep the entire codebase consistent.

- [ ] PR is consistent with the code style guide

## What is the proposed changelog entry for this pull request?

```
*https://oat-sa.atlassian.net/browse/AUT-2332: Update tao skin to add styles for language selector plugin
```

## What changes did you make?

*Give an overview…*

Part 5/6

### CODE REVIEW

File: dev/builder/build.sh [hunk 1]
```diff
@@ -57,7 +57,7 @@ echo "Starting CKBuilder..."
 
 JAVA_ARGS=${ARGS// -t / } # Remove -t from args.
 
-VERSION="4.14.1 TAO-1"
+VERSION="4.15.1 TAO-1"
```
Comment (jsconan) [inline dev/builder/build.sh]: **issue:** The version of CKE must remain in sync with the base repo.

**suggestion:** I'd rather suggest to increase the TAO version
```suggestion
VERSION="4.14.1 TAO-2"
```

**Review by jsconan — CHANGES_REQUESTED** [review]
It looks straightforward.
I have a remark though, about the version. Please see the comment below.

**Review by jsconan — APPROVED** [review]
LGTM

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by shaveko — APPROVED** [review]

**Review by bziondik — APPROVED** [review]

**Pull request merged**